huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -571,209 +589,320 @@ func resourceGaussRedisInstanceV3Update(d *schema.ResourceData, meta interface{}
 			Name: d.Get("name").(string),
 		}
 
-		err := instances.UpdateName(client, d.Id(), updateNameOpts).ExtractErr()
+		err = instances.UpdateName(client, d.Id(), updateNameOpts).ExtractErr()
 		if err != nil {
-			return fmtp.Errorf("Error updating name for huaweicloud_gaussdb_redis_instance %s: %s", d.Id(), err)
+			return diag.Errorf("error updating name for huaweicloud_gaussdb_redis_instance %s: %s", d.Id(), err)
 		}
-
 	}
 
 	if d.HasChange("password") {
 		updatePassOpts := instances.UpdatePassOpts{
 			Password: d.Get("password").(string),
 		}
 
-		err := instances.UpdatePass(client, d.Id(), updatePassOpts).ExtractErr()
+		err = instances.UpdatePass(client, d.Id(), updatePassOpts).ExtractErr()
 		if err != nil {
-			return fmtp.Errorf("Error updating password for huaweicloud_gaussdb_redis_instance %s: %s", d.Id(), err)
+			return diag.Errorf("error updating password for huaweicloud_gaussdb_redis_instance %s: %s",
```

Comment:
remove the word `huawei` in log or error message

Reply:
this PR not fix lint issue